packages/pn-commons/src/components/__test__/CustomDatePicker.test.tsx

```diff
@@ -34,15 +43,15 @@ describe('test CustomDatePicker languages', () => {
     },
     {
       language: 'en',
-      month: 'january',
+      month: 'January',
```

Comment:
Why this change?

Reply:
in hmtl I find the large letter in some months, so I adapted it to html

Reply:
with lowercase as before the test fails?

Reply:
yes, in fact, the tests that failed before were precisely these